src/v/cloud_storage/remote.cc

```diff
@@ -232,28 +232,20 @@ ss::future<upload_result> remote::upload_manifest(
   retry_chain_node& parent) {
     gate_guard guard{_gate};
     retry_chain_node fib(&parent);
+    retry_chain_logger ctxlog(cst_log, fib);
```

Comment:
Maybe there can be one instance of this per module like we do with ss::logger?

Reply:
It's a tiny wrapper around ss::logger and retry_chain_node and the operation that the method performs is usually much longer. For instance, this method uploads object to S3.
In some cases it's a member variable but in others the only option is to create a temporary (for instance here the parent and fib are different on every call).